test/WebSites/RazorPagesClassLibrary/RazorPagesClassLibrary.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <!--  Workaround https://github.com/dotnet/core-setup/issues/3726 -->
     <GenerateDependencyFile>false</GenerateDependencyFile>
+
+    <_EnableAllInclusiveRazorSdk>true</_EnableAllInclusiveRazorSdk>
+    <RazorLangVersion>2.1</RazorLangVersion>
+    <RazorDefaultConfiguration>MVC-2.1</RazorDefaultConfiguration>
```

Comment:
Why target only 2.1 in this project?

Reply:
It's the most recent version of Razor that's available